### PR TITLE
LPS-139100 Raylife D2C Mobile Design for Primary Industry Page

### PR DESCRIPTION
**Pull request by rosig**

### CODE REVIEW

**Comment by rosig** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-139100 Raylife D2C Mobile Design for Primary Industry Page</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-139100">Raylife D2C Mobile Design for Primary Industry Page</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>História</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Média</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Development</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Rosinaldo Guedes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Monica Posin</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

7 files changed, 74 insertions(+), 27 deletions(-)

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/components/fragments/input.scss [hunk 1]
```diff
@@ -90,3 +90,10 @@ input.form-control.search {
 input.hidden {
 	display: none;
 }
+
+.bussiness-type-search-icon {
+	right: 24px;
+	top: 20px;
+	width: 22px !important;
+	height: 22px !important;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/styles/components/fragments/input.scss:98]: @rosig this should be ordered

**Review by kevenleone — CHANGES_REQUESTED** [review]
@rosig just order the classes in .css file

**Comment by kevenleone** [comment]
Great job on that @rosig.

Can you just format??

`gw formatSource` and `yarn format`

Should be enough

**Comment by kevenleone** [comment]
ci:forward

**Review by kevenleone — APPROVED** [review]
Looks great, thank you @rosig !

**Pull request closed without merging**